e2e/benchmarks/benchmark_util.js

```diff
@@ -66,3 +83,120 @@ function generateInput(model) {
     throw e;
   }
 }
+
+/**
+ * Executes the predict function for `model` and times the inference process for
+ * `numRuns` rounds. Then returns a promise that resolves with an array of
+ * inference times for each inference process.
+ *
+ * The inference time contains the time spent by both `predict()` and `data()`
+ * called by tensors in the prediction.
+ *
+ * ```js
+ * const modelUrl =
+ *    'https://tfhub.dev/google/imagenet/mobilenet_v2_140_224/classification/2';
+ * const model = await tf.loadGraphModel(modelUrl, {fromTFHub: true});
+ * const zeros = tf.zeros([1, 224, 224, 3]);
+ * const elapsedTimeArray =
+ *    await profileInferenceTimeForModel(model, zeros, 2);
+ *
+ * console.log(`Elapsed time array: ${elapsedTimeArray}`);
+ * ```
+ *
+ * @param model An instance of tf.GraphModel or tf.LayersModel for timing the
+ *     inference process.
+ * @param numRuns The number of rounds for timing the inference process.
+ */
+async function profileInferenceTimeForModel(model, input, numRuns = 1) {
```

Comment:
looks this method is not used? Do you think the custom model url should use this without the profiling?